src/main/java/org/folio/dao/budget/BudgetExpenseClassPostgresDAO.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.dao.budget;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.json.JsonObject;
+import io.vertx.sqlclient.Tuple;
+import org.folio.rest.jaxrs.model.BudgetExpenseClass;
+import org.folio.rest.persist.DBClient;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.folio.rest.util.ResponseUtils.handleFailure;
+
+public class BudgetExpenseClassPostgresDAO implements BudgetExpenseClassDAO{
```

Comment:
postgres in the name implies that there are other database DAO or in-memory DAO?

Reply:
This is a common practice in acquisition modules. It implies that in the future it will be possible to integrate other databases. At the moment we have only postgres.

Reply:
I think it is okay to stop this practice. Once upon a time, FOLIO used to support mongodb so it made sense then. Up to you if you decide to keep the postgres name.

Reply:
Renamed